src/prof-http/Cargo.toml

```diff
@@ -18,11 +18,14 @@ headers = "0.3.8"
 http = "0.2.8"
 include_dir = "0.7.3"
 once_cell = "1.16.0"
+mappings = "0.5.0"
```

Comment:
This crate replaces `mz-proc`, right? Can you delete `mz-proc` then, since its dead code now?